### DISCUSSION TITLE
What the Containers feature does

### DISCUSSION THREAD

**Discussion by slrslr** [mbnuqw/sidebery#1478] [Q&A]
The Settings / Containers does not show any description how containers works, and how it protects user's privacy. Also the main page of the Github repository and the README is missing any knowledge base/documentation.

I wanted to know if a Sidebery Containers has similar privacy preserving features like https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers/ has: "separates website storage into tab-specific Containers. Cookies downloaded by one Container are not available to other Containers. You can even integrate individual Containers with Mozilla VPN to protect your browsing and location."

**Comment by zelch** [comment]
I believe that the Containers referenced by Sideberry _are_ the multiaccount containers.

At least, when that extension is installed, it most definitely uses them.
